views/users/new.erb

```diff
@@ -0,0 +1,9 @@
+<h1>Please sign up</h1>
+
+<form action="/users" method="post">
+  Email: <input name="email" type="text">
```

Comment:
It's important to mark up your form semantically to make them accessible https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Forms/How_to_structure_an_HTML_form